builders/profiles/hetzner-rx220.nix

```diff
@@ -13,14 +7,6 @@
   boot.supportedFilesystems.zfs = true;
   networking.hostId = "91312b0a";
 
-  # 96G for build roots, 160G for working memory
-  boot.tmp = {
```

Comment:
It would be worth considering in this case having an additional /tmp zfs dataset with sync=disabled: https://github.com/NixOS/infra/blob/main/builders/disk-layouts/efi-zfs-raid0.nix